client/blockchain-service/src/commands.rs

```diff
@@ -61,13 +62,13 @@ pub enum BlockchainServiceCommand {
             tokio::sync::oneshot::Sender<tokio::sync::oneshot::Receiver<Result<(), ApiError>>>,
     },
     QueryFileEarliestVolunteerTick {
-        bsp_id: ProviderId,
+        bsp_id: ChallengeableProviderId,
```

Comment:
Before we just called it `ProviderId`, which for me, seemed to point the the Providers pallet type, meaning that it could be either a BSP or MSP. But in fact the type was referencing the `ProviderId` type used in the Proofs Dealer pallet, which it gets from the `ReadChallengeableProvidersInterface` trait. So it seemed more fitting to call it `ChallengeableProviderId`.